### DISCUSSION TITLE
Consider relaxing tslib dependency or upgrade it.

### DISCUSSION THREAD

**Discussion by OnurGumus** [ReactiveX/rxjs#6686] [General]
Currently rxjs relies on **~2.1.0** for **tslib**. I find this restriction peculiar and also limiting as newer version of tslib is **2.3.1** 

I wonder if we could relax the version constraint to maybe **^2.1.0** or upgrade it to **~2.3.0**?

**Comment by kwonoj** [comment]
Closing the loop, this is in now. https://github.com/ReactiveX/rxjs/pull/6692
